eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/implementation/EndpointStateNotifierBase.java

```diff
@@ -53,15 +53,23 @@ public AmqpEndpointState getCurrentState() {
         return shutdownSignalProcessor;
     }
 
-    void notifyException(ErrorContext error) {
+    void notifyError(ErrorContext error) {
+        Objects.requireNonNull(error);
+
+        logger.asInformational().log("Notify error: {}", error);
```

Comment:
There might be a merge issue here - I'm fairly certain `asInformational` is now `asInfo`, fyi